.github/workflows/test.yml

```diff
@@ -210,25 +178,20 @@ jobs:
           fetch-depth: 0
       - name: Install CIRCT
         uses: ./.github/workflows/install-circt
-      - name: Setup Scala
-        uses: actions/setup-java@v3
+      - name: Install Mill
+        uses: jodersky/setup-mill@master
         with:
-          distribution: 'adopt'
-          java-version: '8'
-          cache: 'sbt'
+          mill-version: 0.11.0
       - name: Setup GPG (for Publish)
         uses: olafurpg/setup-gpg@v3
       - name: Publish
-        run: sbt ci-release
+        run: ./mill -i io.kipp.mill.ci.release.ReleaseModule/publishAll 
```

Comment:
Does this do the right thing for publishing SNAPSHOTs vs. publishing actual releases?

Reply:
Also `publishAll` is almost certainly not right, see the `unipublish` in SBT, we are intentionally only publishing 2 projects, the plugin and unipublish. Unipublish combines the build artifacts of all subprojects into a single jar. There is no support for this built-in (I asked on Gitter) so it will require some manual logic in the mill build just like I had to write in the SBT build.

Reply:
I see, I can custom and rewrite that publish flow in mill.
But I'm out of office this week, will do it next week.